transformations/question_neural_paraphaser/transformation.py

```diff
@@ -0,0 +1,105 @@
+import itertools
+import random
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+import numpy as np
+import torch
+from transformers import T5ForConditionalGeneration, T5Tokenizer
+
+"""
+Base Class for implementing the different input transformations a generation should be robust against.
+"""
+
+
+class DialogNeuralParaphaserPerturbation(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION,
+        TaskType.TEXT_TAGGING,
+    ]
+    languages = ["en"]
```

Comment:
Please add the relevant [keywords](https://github.com/GEM-benchmark/NL-Augmenter/blob/main/docs/keywords.md) too.